src/auth/auth.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Controller, Post, Body } from '@nestjs/common';
+import { AuthService } from './auth.service';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('register')
+  registerUser(
+    @Body('username') username: string,
+    @Body('password') password: string,
+  ) {
+    return this.authService.registerUser(username, password);
+  }
+
+  @Post('login')
+  loginUser(
```

Comment:
Better to use dto's for body or parameters.

Like: 

export class LoginUserDto {
  username: string;
  password: string;
}

https://docs.nestjs.com/controllers#request-payloads

Also read about validation.

https://docs.nestjs.com/techniques/validation

Reply:
registerUser(
    @Body() payload: LoginUserDto
  )

Reply:
Done it, placed in auth.repository and whenever use it import it from there